### PR TITLE
Update configure.md

### PR DESCRIPTION
**Pull request by jboutkab**
fix the CURL command with POST instead of default GET

### CODE REVIEW

**Comment by pedrosousa** [comment]
If you include data in the request via [`-d` or `--data`](https://curl.se/docs/manpage.html#-d), the default verb used by curl will be `POST`. Based on this information, the [curl guidelines](https://developers.cloudflare.com/style-guide/api-content-strategy/guidelines-for-curl-commands/#request-with-json-body-content-post-put-patch) in our Style Guide recommend that you don't add `--request POST` to such curl commands.

However, the default `Content-Type` will be `application/x-www-form-urlencoded`, so you should include a header setting it to `application/json`.

File: content/api-shield/security/sequence-mitigation/configure.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ The response will reflect the rule that has been written with its ID. In case so
 ---
 header: Example using cURL
 ---
-curl "https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation" \
+curl --request POST "https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation" \
```
Comment (patriciasantaana) [inline content/api-shield/security/sequence-mitigation/configure.md]:
```suggestion
curl "https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation" \
```

Comment (jboutkab) [reply content/api-shield/security/sequence-mitigation/configure.md]: Sounds good - I have submitted another PR to fix the endpoints

**Review by patriciasantaana — APPROVED** [review]

File: content/api-shield/security/sequence-mitigation/configure.md (hunk 1 above)
Comment (patriciasantaana) [reply content/api-shield/security/sequence-mitigation/configure.md]: Hey @jboutkab ! did you open a separate PR to fix the endpoints? I don't see it on our opened PRs

**Pull request merged**